pulp/apis/highs_api.py

```diff
@@ -247,32 +246,56 @@ def actualSolve(self, lp, callback=None):
 
     else:
 
+        # Note(maciej): It was surprising to me that higshpy wasn't logging out of the box,
+        #  even with the different logging options set. This callback seems to work, but there
+        #  are probably better ways of doing this ¯\_(ツ)_/¯
+        DEFAULT_CALLBACK = lambda logType, logMsg, callbackValue: print(
+            f"[{logType.name}] {logMsg}"
+        )
+        DEFAULT_CALLBACK_VALUE = ""
+
         def __init__(
             self,
-            mip=True,
```

Comment:
we prefer having the explicit enumeration of parameters, together with a description. This produces the documentation of the APIs in the docs. It's the only way to know in the docs which parameters are supported and which ones are not.
See https://github.com/coin-or/pulp/blob/a017fdbfa2631dfc31bb8d04c50045e1003c94eb/pulp/apis/cplex_api.py#L296-L316